pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -306619,7 +306619,6 @@ self: {
        description = "Efficient implementation of a dependent map with types as keys";
        license = lib.licenses.mpl20;
        hydraPlatforms = lib.platforms.none;
```

Comment:
In practice, you also need to remove the `hydraPlatforms = none` line, since this line prevents Hydra from building this package!

```suggestion
```

Reply:
Hmm, I'm not sure why, but GitHub isn't letting me commit this change from from the UI.  @mpscholten could you commit this as well?

Reply:
Good catch, thanks! Yes will commit and rebase again 👍

Reply:
@cdepillabout just pushed the update 👍